content/guides/component-testing/quickstart-angular.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Quickstart: Angular'
+---
+
+<CtBetaAlert></CtBetaAlert>
+
+To follow along with this guide, you'll need an Angular CLI application.
+
+The quickest way to get started writing component tests for Angular is to use
+the [Angular CLI](https://angular.io/cli).
+
+To create an Angular project:
+
+1. Install the Angular CLI
+
+```bash
+npm install -g @angular/cli
+```
+
+2. Create a new Angular application:
+
+```bash
+ng new my-awesome-app
+```
+
+3. Add Cypress
+
+```bash
+npm install cypress -D
```

Comment:
not sure if you want to instruct them to `cd my-awesome-app` or not. Your call.

Reply:
Good point. If you do update it the react and vue guides should get it as well.